src/orbit.rs

```diff
@@ -212,8 +212,8 @@ pub async fn create_orbit(
     };
 
     // fails if DIR exists, this is Create, not Open
-    let chunk_store = ChunkStore::new(config.storage.chunks.clone());
-    if chunk_store.exists(id.get_cid()).await? {
+    let storage_utils = StorageUtils::new(config.storage.blocks.clone());
```

Comment:
How would you feel about having a method on the config struct to create the storage struct, e.g. `config.storage_utils()`

Reply:
I think it's more idiomatic to build something by passing it its configuration, than building something from a configuration object (unless it's a builder). But fundamentally it wouldn't change much.

I don't think it's worth our time to change it though, because the plan is to have that object created only once, I just wanted to limit my changes.